packages/core/src/resources/RepositoryFiles.ts

```diff
@@ -109,12 +109,13 @@ export class RepositoryFiles<C extends boolean = false> extends BaseResource<C>
     );
   }
 
-  showRaw(projectId: string | number, filePath: string, ref: string, options?: Sudo) {
+  showRaw(projectId: string | number, filePath: string, options?: BaseRequestOptions) {
     const [pId, path] = [projectId, filePath].map(encodeURIComponent);
 
-    return RequestHelper.get()(this, `projects/${pId}/repository/files/${path}/raw`, {
-      ref,
-      ...options,
-    }) as unknown as Promise<Blob>;
+    return RequestHelper.get()(
+      this,
+      `projects/${pId}/repository/files/${path}/raw`,
+      options,
```

Comment:
@jdalrymple Could you check if this is ok? I am quite sure that ref isn't mandatory, but I don't know if options still needs the spread syntax.

Reply:
Youre right! https://docs.gitlab.com/ee/api/repository_files.html#get-raw-file-from-repository